agent/engine/engine_unix_integ_test.go

```diff
@@ -1561,28 +1640,94 @@ func TestExecCommandAgent(t *testing.T) {
 	verifyContainerRunningStateChange(t, taskEngine)
 	verifyTaskRunningStateChange(t, taskEngine)
 
-	client, err := sdkClient.NewClientWithOpts(sdkClient.WithHost(endpoint), sdkClient.WithVersion(sdkclientfactory.GetDefaultVersion().String()))
-	require.NoError(t, err, "Creating go docker client failed")
-
 	ctx, cancel := context.WithCancel(context.TODO())
 	defer cancel()
 
 	containerMap, _ := taskEngine.(*DockerTaskEngine).state.ContainerMapByArn(testTask.Arn)
 	cid := containerMap[testTask.Containers[0].Name].DockerID
-	inspectState, _ := client.ContainerInspect(ctx, cid)
 
+	verifyExecCmdAgentExpectedMounts(t, ctx, client, testTaskId, cid, testContainerName, testExecCmdHostBinDir)
+	pid1 := verifyMockExecCommandAgentIsRunning(t, client, cid)
+	seelog.Infof("Verified mock ExecCommandAgent is running (pid1=%s)", pid1)
+	killMockExecCommandAgent(t, client, cid, pid1)
+	seelog.Infof("kill signal sent to ExecCommandAgent (pid1=%s)", pid1)
```

Comment:
are these logs necessary? would prefer them removed

Reply:
I figured if something goes wrong with this test, these logs can help figure out what happened, since integ tests are kinda real life